client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -111,3 +113,20 @@ def test_get_access_token(self, client):
         token = endpoint.get_access_token()
 
         assert token is None
+
+    def test_get_deployed_model(self, client, experiment_run, model_for_deployment):
+        model = model_for_deployment['model'].fit(
+            model_for_deployment['train_features'],
+            model_for_deployment['train_targets'],
+        )
+        experiment_run.log_model_for_deployment(model)
```

Comment:
`log_model_for_deployment()` is deprecated.
```suggestion
        experiment_run.log_model(model, custom_modules=[])
        experiment_run.log_requirements(['scikit-learn'])
```